### ISSUE TITLE
MiKTeX Console not accessable after updating

### ISSUE THREAD

**Issue by BeerStud** [MiKTeX/miktex#111]
After updating MiKTeX, I now cannot access the console or any of the other MiKTeX programs. This is the error message shown when attempting to access the console "C:\Program Files\MiKTeX 2.9\miktex\bin\x64\miktex-console.exe":

MiKTeX Problem Report
Message: Windows API error 87: The parameter is incorrect.

Data:
path="C:\ProgramData\MiKTeX\2.9\miktex/data/le\1756f6ade37d5f870f23c05a7f81e257.fndb"
Source: Libraries\MiKTeX\Core\MemoryMappedFile\win\winMemoryMappedFile.cpp
Line: 83

Since I cannot access the console, I cannot update any of the programs. In addition, TeXstudio has stopped working with "pdflatex.exe". The program simply hangs.

What can I do, other than completely removing MiKTeX and starting over to correct this problem?

This is happening on a Windows 10 PRO amd64 bit machine.

**Comment by edocevoli** [comment]
Stop all other (MiKTeX) programs before starting the console.

**Comment by BeerStud** [comment]
There are no other programs running. This started only after the update a few days ago.

**Comment by edocevoli** [comment]
Then it is a system-dependent problem. Since I cannot reproduce the issue, it is not possible to provide a fix.

**Comment by BeerStud** [comment]
I just emptied all the "tmp" folders and rebooted my system. I can now access the console. I tried this before, but it failed to alleviate the problem. I don't know what has changed. However, "TxTstudio" still fails with "pdflatex.exe". It just hangs when attempting to compile a file. This has only occurred after the update on I believe April 11. How can I revert the updated files to an earlier version? I believe that there is something inherently wrong with the new version of "pdflatex" and perhaps other files.
